### PR TITLE
Add key without value check

### LINKED ISSUE #7: Add check: Keys without values
Example:

```env
# .env

# Wrong
RAILS_ENV

# Correct
RAILS_ENV=

# Correct
RAILS_ENV=development
```

This check should display a message:
> .env:1 The RAILS_ENV key should be with a value

### PR DESCRIPTION
**Pull request by mstruebing**
closes #7

### CODE REVIEW

**Comment by mstruebing** [comment]
Thank you, it was a pleasure :)
I've fixed your suggestions and added the check also to the readme, I also removed the link to the issue (I've thought this would be want you want)

File: src/checks/key_without_value.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        Self {
+            warning: Warning::new("The {} key should be with a value or have an equal sign"),
+        }
+    }
+}
+
+impl Check for KeyWithoutValueChecker {
+    fn run(&self, line: &LineEntry) -> Option<Warning> {
+        if !line.raw_string.contains('=') {
+            Some(self.warning.clone())
```
Comment (mgrachev) [inline src/checks/key_without_value.rs]: You should find a key and substitute it in a message to get the message like that: `The RAILES_ENV key should be with a value or have an equal sign`

Comment (mstruebing) [reply src/checks/key_without_value.rs]: Fixed. :)

**Comment by mstruebing** [comment]
:tada:

**Comment by mgrachev** [comment]
@mstruebing Thank you very much! You're awesome! ❤️

**Pull request merged**